lib/Data/Content/CreateContentDraftData.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\RepositoryForms\Data\Content;
+
+use eZ\Publish\API\Repository\Values\ValueObject;
+
+class CreateContentDraftData extends ValueObject
+{
+    public $contentId;
```

Comment:
@bdunogier We could maybe also "wrap" a readonly VO in a writeable VO for the purpose of the mapper, but again, does this make sense to do?